test/IDE/complete_some_any.swift

```diff
@@ -0,0 +1,26 @@
+// RUN: %batch-code-completion
+
+func test1(x: #^PARAM?check=HAS_SOME_ANY;check=HAS_REPEAT^#) {}
+// FIXME: 'repeat' is not valid here semantically but we allow it syntactically and can't (easily) tell in the parser whether 'repeat' is valid. Not worth fixing in the old parser.
```

Comment:
I might be misunderstanding, but can't you have `repeat` here? e.g you could have:

```swift
func test1<each T>(x: repeat each T) {}
```

Or do you mean it can't be valid semantically because we can tell there's no pack in scope?

Reply:
Oh, you’re right.